crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -36,19 +36,37 @@ async fn local_node_mixed_limit_and_market_orders() {
 }
 
 /// The block number from which we will fetch state for the forked tests.
-pub const FORK_BLOCK: u64 = 18477910;
-/// USDC whale address as per [FORK_BLOCK].
-pub const USDC_WHALE: H160 = H160(hex_literal::hex!(
+pub const FORK_BLOCK_MAINNET: u64 = 18477910;
+/// USDC whale address as per [FORK_BLOCK_MAINNET].
+pub const USDC_WHALE_MAINNET: H160 = H160(hex_literal::hex!(
     "28c6c06298d514db089934071355e5743bf21d60"
 ));
 
 #[tokio::test]
 #[ignore]
-async fn forked_node_single_limit_order_mainnet() {
+async fn forked_node_mainnet_single_limit_order() {
     run_forked_test_with_block_number(
-        forked_single_limit_order_test,
-        std::env::var("FORK_URL").expect("FORK_URL must be set to run forked tests"),
-        FORK_BLOCK,
+        forked_mainnet_single_limit_order_test,
+        std::env::var("FORK_URL_MAINNET")
+            .expect("FORK_URL_MAINNET must be set to run forked tests"),
+        FORK_BLOCK_MAINNET,
+    )
+    .await;
+}
+
+pub const FORK_BLOCK_GNOSIS: u64 = 32070725;
+/// USDC whale address as per [FORK_BLOCK_GNOSIS].
+pub const USDC_WHALE_GNOSIS: H160 = H160(hex_literal::hex!(
```

Comment:
nit: Is there a safer address (this seems to be the balancer vault which will likely be phased out soon with the launch of v3)? We could also use wxDAI if it's easier to find a long term whale there.

Reply:
Because the fork is pinned to the above block, this whale address will always be fine. If the fork block changes, we might need to change it of course.